opendaq/Opc.Ua.Daq.Bsp.Types.bsd

```diff
@@ -57,6 +57,12 @@
         <opc:Field TypeName="opc:Int32" Name="NoOfParameters"/>
         <opc:Field LengthField="NoOfParameters" TypeName="ns1:DaqKeyValuePair" Name="Parameters"/>
     </opc:StructuredType>
+    <opc:StructuredType BaseType="tns:BaseRuleDescriptionStructure" Name="ExplicitDomainRuleDescriptionStructure">
+        <opc:Documentation>It should be used only if the "MinExepctedDelta" or "MaxExpectedDelta" is needed</opc:Documentation>
```

Comment:
Considering we added both min and max, the description should say that if only one is needed, the other should be set to 0. Besides that this lgtm.

Reply:
Add a new commit to fix it.